module/metrics/server.go

```diff
@@ -20,14 +20,12 @@ type Server struct {
 
 // NewServer creates a new server that will start on the specified port,
 // and responds to only the `/metrics` endpoint
-func NewServer(log zerolog.Logger, port uint, enableProfilerEndpoint bool) *Server {
+func NewServer(log zerolog.Logger, port uint) *Server {
 	addr := ":" + strconv.Itoa(int(port))
 
 	mux := http.NewServeMux()
 	mux.Handle("/metrics", promhttp.Handler())
-	if enableProfilerEndpoint {
-		mux.Handle("/debug/pprof/", http.DefaultServeMux)
-	}
+	mux.Handle("/debug/pprof/", http.DefaultServeMux)
```

Comment:
I don’t find a way to dynamically enable/disable this endpoint. So I ended up keep it enabled. Thoughts?

Reply:
Who has access to the port? Is it only internal? Otherwise, this could be a kind of attack vector?

Reply:
Reverted